agent/api/task/service_connect.go

```diff
@@ -28,10 +28,6 @@ type ServiceConnectConfig struct {
 type RuntimeConfig struct {
 	// Host path for the administration socket
 	AdminSocketPath string `json:"adminSocketPath"`
-	// HTTP Path + Params to get statistical information
-	StatsRequest string `json:"statsRequest"`
-	// HTTP Path + Params to drain ServiceConnect connections
-	DrainRequest string `json:"drainRequest"`
```

Comment:
Since you are already using AdminSocketPath, using the request from here is more sane allowing for potentially different values over time which forcing them in the client doesn't allow. I would leave the requests here

Reply:
> allowing for potentially different values over time

hmm, you  mean different paths for for different tasks? 

As I'm seeing it, the Appnet client should know how to talk to Appnet agent. Otherwise it's not really the Appnet client, but rather a generic http-over-uds client. (Which I'm not necessarily opposed to).

Reply:
The integration of AppNet agent container and versions of that should be consolidated. Different versions could change the http paths over time. So in runtimeConfig it is serialized to be picked up by new Agent versions but in the client it would need another mechanism to know which task is running which AppNetAgent etc. The client is still valuable in that it abstracts away the UDS HTTP client even if it gets the URL from a runtimeConfig or statically. That distinction doesn't change the abstraction or value of said abstraction

Reply:
https://github.com/aws/amazon-ecs-agent/pull/3230 as the differences, shows that it isn't really a change to what the client knows or doesn't know. But it changes where the integration configured values are stored together.